wfe-core/src/main/java/ru/runa/wfe/commons/Utils.java

```diff
@@ -223,6 +223,16 @@ public static String getReceiveMessageNodeSelector(VariableProvider variableProv
         Collections.sort(selectors);
         return Joiner.on(MESSAGE_SELECTOR_DELIMITER).join(selectors);
     }
+    
+    public static Map<String, String> getVariableNameMappedNamePair(BaseMessageNode messageNode, String variableName) {
+        Map<String, String> map = Maps.newHashMap();
+        for (VariableMapping mapping : messageNode.getVariableMappings()) {
+            if (mapping.isPropertySelector() && mapping.getMappedName().contains(variableName)) {
```

Comment:
mapping.getMappedName().contains(variableName) -> Objects.equal(mapping.getMappedName(), variableName)

Reply:
mappedName stored like "${Variable1}" but property Name of Variable object that is passed in function as variableName stored like "Variable1", so Objects.equal will always return false. Left .contains as it was.